src/current/cockroachcloud/stream-changefeed-to-snowflake-aws.md

```diff
@@ -152,13 +138,13 @@ If your changefeed is running but data is not displaying in your S3 bucket, you
 
 1. Log in to Snowflake as a user with [read and write access](https://docs.snowflake.net/manuals/user-guide/security-access-control-overview.html) to a cluster.
 
-1. Navigate to the **Worksheet** view.
+1. Navigate to the **Worksheets** and select a worksheet.
```

Comment:
```suggestion
1. Navigate to the **Worksheets** page and select a worksheet.
```

Reply:
Snowflake calls it a page here https://docs.snowflake.com/en/user-guide/ui-worksheet but we could also say Navigate to **Worksheets**

Reply:
I'm not sure why we would want to change this when it is called the Worksheets page?

Reply:
apologies, I meant either calling it 1. the **Worksheets** page or 2. Navigate to **Worksheets** instead of saying "Navigate to the **Worksheets**..."